app/frontend/tests/unit/views/form/Success.spec.js

```diff
@@ -4,6 +4,15 @@ import Success from '@/views/form/Success.vue';
 
 const localVue = createLocalVue();
 
+jest.mock('@/utils/permissionUtils', () => (
+  {
+    ...(jest.requireActual('@/utils/permissionUtils')),
+    determineFormNeedsAuth: jest.fn()
+  }
```

Comment:
Normally we import the component first, then do the jest.mock autowire for the same component, not the other way around.

Reply:
Don't know the syntax but determineFormNeedsAuth is an exported function and doing it any way other than this (tried a million other layouts) failed 🤷

Reply:
Kk - if there's some strange jest load ordering issue going on it's fine to leave as is.